<div class="container aboutBox">
	<div class="row">
		
	<div class="col-md-offset-1 col-md-11">
<h1>About Cistrome DB resources</h1>
<p>Cistrome DB is a comprehensive resource of hg38 and mm10 ChIP-seq data collection.
Here is a brief introduction about the workflow of <a href="http://cistrome.org/chilin">ChiLin</a>.
</p>
<ul>
mapping step: 
    bwa aln -q 5 -l 32 -k 2 -t 8 index FASTQ > sai
    bwa samse index sai FASTQ > sam  
</ul>
<ul>
reads filter step: samtools view -bS -t chromInfo_file.txt -q 1 sam > bam  
</ul>
<ul>
(optional)sort bam: samtools sort -m 4000000000 bam > bam_sort  
</ul>
<ul>
(optional)collect statistics: samtools flagstat bam > bam.stat  
</ul>  
<ul>
 Peak calling: macs2 callpeak --SPMR -B -q 0.01 --keep-dup 1 --extsize=146 --nomodel -g hs   -t bam  -n test  
</ul>
<ul>
 bigwiggle generation: 
    bedtools intersect -a bedGraph -b chrom.bed -wa -f 1.00 > bedGraph.tmp
    bedGraphToBigWig bedGraph.tmp chromInfo_file.txt bigwiggle
</ul>
<ul>
choose the top 5k summits for motif scan, use all summits if the number of summits is less than 5k: 
    MDSeqPos.py -d  -w 600  -p 0.001 -m cistrome.xml -O output bed hg38  
</ul>  
<ul>
choose the top 10k peaks for RP computing: RegPotential.py -t top -g refGene -n test -d 100000  
</ul>  
</div>

<div class="col-md-offset-1 col-md-11">
<h1>Cistrome DB function</h1>
<p>The main cistrome performs data search function with 4 views and 1 inspector:</p>
<li>
	Quality control view
</li>
<li>
	Motif ScanView
</li>
<li>
	Regulatory Potential View
</li>
<li>
	Similar ChIP-seq dataset view
</li>
<h1>CistromeDB Pulication Policies and Guidelines</h1>
<p>
The Cistrome DB program has established the following policy to clarify freedom to publish or present on findings using Cistrome DB data. Data users may freely download and analyze CistromeDB data with the following restrictions:
<ol type="1">
	<li>
		We are now working on a global integrative analysis study based on Cistrome DB.
	</li>
<!-- 	<li>
		No batch web scraping is allowed.
	</li> -->
	<li>
		<p>Users are not allowed to download more than 100 datasets.
	</li>
	<li>
	<p>Data users should properly cite the Cistrome Project in any publication, please use the following two publications temporarily:.</p>
	<ol type="A">
<li>Sun H, Qin B, Liu T, Wang Q, Liu J, et al. (2013) Data and Text Mining CistromeFinder for ChIP-seq and DNase-seq data reuse. Bioinformatics: 1–3.	</li>
<li>Qin B, Zhou M, Ge Y, Taing L, Liu T, et al. (2012) CistromeMap: a knowledgebase and web server for ChIP-Seq and DNase-Seq studies in mouse and human. Bioinformatics 28: 1411–1412. </li>
	</ol>
     </li>
</ol>
</div>

<div class="col-md-offset-1 col-md-11">
<h1>People and Contacts</h1>

<p>The CistromeDB is coordinated by a project team comprised of individuals from both the Dana Farber Cancer Institute and Tongji University.  CistromeDB is still under development. If you have any question and suggestion,  do not hesitate to contact us ! </p>
<footer>DC team: cistrome.dev@gmail.com.</footer>
</div>	
</div>
</div>


